src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+Wallet.kt

```diff
@@ -162,9 +171,14 @@ internal fun TradingStateMachine.receivedFills(
     payload: Map<String, Any>,
     subaccountNumber: Int,
 ): StateChanges {
-    val size = parser.asList(payload["fills"])?.size ?: 0
+    val fills = parser.asList(payload["fills"])
+    val size = fills?.size ?: 0
     return if (size > 0) {
-        wallet = walletProcessor.receivedFills(wallet, payload, subaccountNumber)
+        wallet = walletProcessor.receivedFillsDeprecated(wallet, payload, subaccountNumber)
```

Comment:
this can be in an `else` block right?

Reply:
actually nevermind, this PR seems to only affect write path, not read path. so still need to write to the old map if i am understanding correctly.

Reply:
Read should go through internal state as well.. but I don't think there is read on fills within Abacus